src/vdom/remove.js

```diff
@@ -0,0 +1,3 @@
+export default function remove(el) {
+	el.parentNode && el.parentNode.removeChild(el);
```

Comment:
There are a few of these cases now in the codebase so the gzipped size should be fine with the explicit null check.  I generally try to benchmark to determine where it's worthwhile, but I'm fairly certain in this case it would be since it's immediately accessing `.removeChild` on the object value.